### DISCUSSION TITLE
ScriptReference/GUI.GetNameOfFocusedControl

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#191] [Notes]
# ScriptReference/GUI.GetNameOfFocusedControl

https://docs.unity3d.com/ScriptReference/GUI.GetNameOfFocusedControl.html

**Comment by mvi** [comment]
Note that handles can also return a valid name when being manipulated. For example, while interacting with Unity's built in translation handle `GetNameOfFocusedControl()` will report the axis being manipulated ("xAxis", "yAxis" or "zAxis")
